scripts/update-manpages

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# =============
+# = Functions =
+# =============
+
+cleanup() {
+        printf '👋  Change back to original directory “%s”\n' "$CURRENT_DIR"
+        cd "$CURRENT_DIR" || exit
+}
+
+# ========
+# = Init =
+# ========
+
+CURRENT_DIR="$PWD"
+SCRIPTS_DIR=$(dirname "$0")
+. "$SCRIPTS_DIR/realpath"
```

Comment:
why not directly execute include-common, it already sets some of the needed vars.